Zombie/src/main/java/dk/gruppeseks/bodtrd/zombie/ZombieProcessor.java

```diff
@@ -133,4 +139,49 @@ private void attackPlayer(World world, Entity zombie, Entity player)
             zWep.setAttackCooldown(zWep.getAttackSpeed());
         }
     }
+
+    private void rayCast(Entity zombie)
```

Comment:
imo this funcitonality is misplaced, shouldnt it be AI in conjunction with collision that takes care of determining collision in the form of raycasts?